workflows/src/main/kotlin/com/r3/developers/csdetemplate/utxoexample/workflows/TestContractFlow.kt

```diff
@@ -284,15 +284,15 @@ class TestContractFlow: RPCStartableFlow  {
 
                 val txBuilder = ledgerService.getTransactionBuilder()
                     .setNotary(Party(notary.name, notaryKey))
-                    .setTimeWindowBetween(Instant.now(), Instant.now().plusMillis(1.days.toMillis()))
+                    .setTimeWindowBetween(Instant.now(), Instant.now().plusMillis(Duration.ofDays(1).toMillis()))
                     .addInputState(inputStateRef)
                     .addInputState(inputStateRef)
                     .addOutputState(chatState)
                     .addCommand(ChatContract.Update())
                     .addSignatories(chatState.participants)
```

Comment:
this code repeats many times ...
maybe worth having a shorter function to call (private and local to the test class)

Reply:
I played around with abstracting this out a bit, but settled on this format. It doesn't quite repeat, its setting up the different test cases for the contract, so it needs to be visible. The answer here is to build the Corda 5 equivalent of the Leedger DSL. which had a ver neat way of expressing the transaction which needed to be created for the test.